CoreScriptsRoot/Modules/Settings/Utility.lua

```diff
@@ -1130,25 +1243,46 @@ local function CreateSelector(selectionStringTable, startPosition)
 end
 
 local function ShowAlert(alertMessage, okButtonText, settingsHub, okPressedFunc, hasBackground)
-	if CoreGui.RobloxGui:FindFirstChild("AlertViewFullScreen") then return end
+	local parent = CoreGui.RobloxGui
+	local AlertViewBacking = nil
```

Comment:
That's how I found it! It looks like all of the "important" layout gui objects are like this.